deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -1,9 +1,20 @@
 ---
 version: "3.7"
 
+volumes:
+  traefik-certs: null
+  ocis-settings: null
+  ocis-store: null
+  ocis-storage-system: null
+  ocis-storage-users: null
+  ocis-storage-sharing: null
+
+networks:
+  ocis-net:
+
 services:
   traefik:
-    image: traefik:v2.5
+    image: traefik:v2.6.6
```

Comment:
we should also keep this consistent on the other deployment examples in a follow up PR